page.codeberg.JakobDev.jdProcessFileWatcher.yaml

```diff
@@ -0,0 +1,72 @@
+id: page.codeberg.JakobDev.jdProcessFileWatcher
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: jdprocessfilewatcher
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_PYWEBENGINE=1
+finish-args:
+  - --filesystem=xdg-data/flatpak/exports/share/applications:ro
+  - --filesystem=/var/lib/flatpak/exports/share/applications:ro
+  - --filesystem=xdg-data/flatpak/exports/share/icons:ro
+  - --filesystem=xdg-data/flatpak/exports/share/icons:ro
+  - --filesystem=xdg-data/flatpak/runtime:ro
+  - --filesystem=/var/lib/flatpak/runtime:ro
+  - --filesystem=/var/lib/flatpak/runtime:ro
+  - --filesystem=xdg-data/applications:ro
+  - --filesystem=xdg-data/flatpak/app:ro
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=/var/lib/flatpak/app:ro
+  - --filesystem=host-etc:ro
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: strace
```

Comment:
strace can't run in the sandbox unless `--devel`

Reply:
strace isn't run in the sandbox. It is run on the Host. `--devel` don't help here.

Reply:
then you shouldn't be building it.

Reply:
Why?

Reply:
how do you guarantee that this will run on the host and how do you run it on the host.

Reply:
It is run using `ld-linux.so.2` with the `--libary-path` argument, so it will only use libs from the Runtime and the App. You can find the full logic [here](https://codeberg.org/JakobDev/jdProcessFileWatcher/src/commit/bad7b1b0ebb22ac926a6f3668d83764591556f37/jdProcessFileWatcher/StraceWrapper.py#L128). I have tested it on an older Ubuntu and it works.